### ISSUE TITLE
WCAG 2.2 1.4.6 Contrast (Enhanced) failure in Tr stylesheet

### ISSUE THREAD

**Issue by plehegar** [w3c/tr-pages#107]
We should consider following [Success Criterion 1.4.6 Contrast (Enhanced)](https://www.w3.org/TR/WCAG22/#contrast-enhanced).

from @dd8:
[[
The word NOTE in the note referring to Guideline 1.4 in 1.3.3 has contrast ratio 4.58
https://www.w3.org/TR/WCAG22/#sensory-characteristics

It's 12pt with text color: rgb(23,130,23) and background-color: rgb(233,251,233) and font-weight: 400 (normal)
https://contrast-ratio.com/#rgb%2823%2C130%2C23%29-on-rgb%28233%2C251%2C233
]]

from @patricklauke:
[[
I suspect this is an upstream issue on the generic W3C styles for all specs. changing the foreground colour for .note::before, .note > .marker, details.note > summary  / --noteheading-text in base.css from hsl(120, 70%, 30%) to hsl(120, 70%, 22%) will bump the contrast against #E9FBE9 to 7.25:1
]]

Originally from https://github.com/w3c/wcag/issues/3105
